capstone/src/main/webapp/youtubeScript.js

```diff
@@ -23,29 +23,41 @@ const ATTRIBUTES_BY_LANGUAGE = {
 
 /** Calls youtube servlet and passes output to perspctive */
 async function callYoutube() {
-  const channelId = document.getElementById('channelIdForAnalysis').value;
+  const channelId = document.getElementById('channelIdForAnalysis').value.replace(/ /g, '');
   if (!channelId) {
     return;
   }
   var response;
-  if (channelId[0] == "U" && channelId[1] == "C" && channelId.length == 24) {
+  if (channelId[0] == "U" && channelId[1] == "C" && channelId.length == 24 && isLetter(channelId[channelId.length-1])) {
```

Comment:
I'd suggest adding a comment explaining what this is checking

Reply:
addressed in 'explain channel ID manipulation and remove semicolons"